demo/api/src/app.module.ts

```diff
@@ -89,8 +89,10 @@ export class AppModule {
                         },
                         context: ({ req }: { req: Request }) => ({ ...req }),
                         cors: {
-                            credentials: true,
-                            origin: config.corsAllowedOrigins.map((val: string) => new RegExp(val)),
+                            origin: config.corsAllowedOrigin,
+                            methods: ["GET", "POST"],
+                            credentials: false,
```

Comment:
This probably should still be true so block loader requests in the block preview work. However, I can't verify this currently since the requests are broken because of the auth proxy:

<img width="1920" height="958" alt="Image" src="https://github.com/user-attachments/assets/66bf2f87-4353-4f02-b4b9-c8ead0a22275" />

Both issues should be resolved once we have a BFF for the block loader requests. Then `credentials: false` should be alright.